components/homme/src/share/prim_driver_base.F90

```diff
@@ -1406,12 +1336,24 @@ subroutine applyCAMforcing_tracers(elem,hvcoord,np1,np1_qdp,dt,adjustment)
   ! phi/density being diagnostic.  theta model NH could do the conversion constant 
   ! density which would simplify this routine
   !
-  use control_mod,        only : use_moisture
+  ! NOTE about ps_v/dp3d
+  ! init:
+  !   (both ps_v and dp3d are valid)
+  ! do: 
+  !    physics  (uses ps_v to compute pressure levels. doesn't change ps_v)
+  !    applyCAMforcing_tracers  use ps_v for initial pressure.  
+  !                             may adjust dp3d for mass conservation (if adjust_ps=.false.)
+  !                             ps_v no longer valid
+  !    dynamaics                should only use dp3d
```

Comment:
minor comment, "dynamics" is misspelled here.

Reply:
fixed